README.rst

```diff
@@ -126,10 +126,28 @@ A single ``else`` block may be present, but ``elif`` is not supported.
 
 See also `pyupgrade’s similar feature <https://github.com/asottile/pyupgrade/#python2-and-old-python3x-blocks>`__ that removes outdated code from checks on the Python version.
 
+Django 1.7
+-----------
+
+`Release Notes <https://docs.djangoproject.com/en/dev/releases/1.7/>`__
```

Comment:
slightly better to link to `en/stable/releases` so the big yellow "unsupported version" banner isn't present

Reply:
Oh great, I was looking for something like that. Good to know!